SimMuon/RPCDigitizer/python/customizeRPCDigi.py

```diff
@@ -1,152 +0,0 @@
-    process.muonDigi = cms.Sequence(
-        process.simMuonCSCDigis + 
-        process.simMuonDTDigis + 
-        process.simMuonRPCDigis + 
-        process.simMuonGEMDigis + 
-        process.simMuonGEMPadDigis
-    )
-    process.pdigi = cms.Sequence(
-        cms.SequencePlaceholder("randomEngineStateProducer")*
-        cms.SequencePlaceholder("mix")*
-        process.muonDigi
-    )
-    process = append_GEMDigi_event(process)
-    return process
-
-# Customizations for the background
-def customize_digi_noRPCbkg(process):
-    process.simMuonRPCDigis.doBkgNoise = False
-    return process
-
-
-
-
-# adding re-digi costumisation - to be used for dedicated trigger studies
-def customise_rpcRedigi(process):
-    process.load('Configuration.StandardSequences.Digi_cff')
-    process.simMuonRPCReDigis = process.simMuonRPCDigis.clone()
-    process.simMuonRPCReDigis.digiRPCModelConfig = cms.PSet(
-        Frate = cms.double(1.0),
-        Gate = cms.double(1.0),
-        IRPC_electronics_jitter = cms.double(0.1),
-        IRPC_time_resolution = cms.double(1.0),
-        Nbxing = cms.int32(799),
-        BX_range = cms.int32(400),
-        Rate = cms.double(0.0),
-        averageClusterSize = cms.double(1.5),
-        averageEfficiency = cms.double(0.95),
-        cosmics = cms.bool(False),
-        deltatimeAdjacentStrip = cms.double(3.0),
-        linkGateWidth = cms.double(1.0),
-        printOutDigitizer = cms.bool(False),
-        signalPropagationSpeed = cms.double(0.66),
-        timeJitter = cms.double(1.0),
-        timeResolution = cms.double(2.5),
-        timingRPCOffset = cms.double(50.0)
-        ),
-    process.simMuonRPCReDigis.digiIRPCModelConfig = cms.PSet(
-        Frate = cms.double(1.0),
-        Gate = cms.double(1.0),
-        IRPC_electronics_jitter = cms.double(0.1),
-        IRPC_time_resolution = cms.double(1.0),
-        Nbxing = cms.int32(799),
-        BX_range = cms.int32(400),
-        Rate = cms.double(0.0),
-        averageClusterSize = cms.double(1.5),
-        averageEfficiency = cms.double(0.95),
-        cosmics = cms.bool(False),
-        deltatimeAdjacentStrip = cms.double(3.0),
-        linkGateWidth = cms.double(1.0),
-        printOutDigitizer = cms.bool(False),
-        signalPropagationSpeed = cms.double(0.66),
-        timeJitter = cms.double(1.0),
-        timeResolution = cms.double(2.5),
-        timingRPCOffset = cms.double(50.0)
-        )
-    process.RandomNumberGeneratorService.simMuonRPCReDigis = cms.PSet(
-        initialSeed = cms.untracked.uint32(13579),
-        engineName = cms.untracked.string('TRandom3')
-        )
-    process.rpcRecHits.rpcDigiLabel = cms.InputTag("simMuonRPCReDigis")
-    process.validationMuonRPCDigis.rpcDigiTag = cms.untracked.InputTag("simMuonRPCReDigis")
-    process.reconstruction_step.replace(
-        process.rpcRecHits,
-        cms.Sequence(process.simMuonRPCReDigis+process.rpcRecHits)
-        )
-    return process
```

Comment:
Not by accident. It is now included in SimMuon/Configuration/python/customizeMuonDigi.py which will contain all expert customizations for muon digi studies.